examples/matrix_t_test.html

```diff
@@ -13,6 +13,7 @@
                 var m_u8 = new Module.matrix_t(2, 2, 0x0100 | 0x01, data_u8);
                 console.log(m_u8);
                 console.log(m_u8.data);
+                console.log(Module.matrix_t.get(m_u8));
```

Comment:
The static method can not hold the Uint8Array. 
The `data` entry return only an empty Uint8Array instead of a filled one. I will try to understand why.

Reply:
see issue https://github.com/kalwalt/webarkit-jsfeat-cpp/issues/6